### PR TITLE
Remove old nix setup: `mv ./__std__ -> ./nix`

### PR DESCRIPTION
**Pull request by zeme-iohk**
Old hydra CI now always fail as it won't find the `release.nix` 
New hydra CI (the `plutus-flaked-*` in https://hydra.iohk.io/project/Cardano) hasn't triggered in a few PRs...
So we now rely fully on Cicero

### CODE REVIEW

File: nix/cells/plutus/packages/check-the-flake.nix [hunk 1]
```diff
@@ -29,10 +27,10 @@ cell.library.pkgs.writeShellApplication {
 
     derivation_fragments=$(
       find \
-        "$(repo-root)/__std__/cells" \
+        "$(repo-root)/nix/cells" \
         -name "*.nix" \
         -and -not -name "*default.nix" \
-        -and -path "*packages*" \
+        -and -path "*/packages*" \
```
Comment (zeme-iohk) [inline nix/cells/plutus/packages/check-the-flake.nix:33]: was causing `library/agda-packages` to sneak in...

File: nix/README.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ A description of the arguments follows:
   Provides access to the cell's blocks.\
   This is a shorthand for `inputs.cells.<cell>.<cell-block>`, where `<cell>` evaluates to the cell housing this nix file.\
   Examples:
-  - `cell.library.agda-packages` only works for code in `/cells/plutus`\
+  - `cell.library.agda-packages` only works for code in `./nix/cells/plutus`\
```
Comment (michaelpj) [inline nix/README.md:42]: these links now look wrong? they look like relative links but the file is in `nix` already. `/nix` could look like a link from the root, but `./nix` looks wrong. I'd just leave them relative and leave the nix bit off.

File: nix/README.md [hunk 3]
```diff
@@ -111,7 +111,5 @@ As another example, consider the file `__std__/cells/plutus/library/default.nix`
 
 Migration to `std` will be 100% complete once all these are done:
```
Comment (michaelpj) [inline nix/README.md:112]: Do you plan to do all these here, or are these still todos?

**Review by michaelpj — APPROVED** [review]

File: nix/README.md (hunk 3 above)
Comment (zeme-iohk) [reply nix/README.md:112]: I'll spread those out into multiple PRs. 
Maybe I've learned to keep my PRs short :P

File: nix/README.md (hunk 2 above)
Comment (zeme-iohk) [reply nix/README.md:42]: 👍 will do

**Comment by michaelpj** [comment]
Where are we with this? can you reproduce the failure locally?

**Comment by zeme-iohk** [comment]
> Where are we with this? can you reproduce the failure locally?

Trying the right now, though I'm on darwin. 
Also pushed a noop change to trigger CI -> maybe the logs have come back

**Comment by zeme-iohk** [comment]
```
Preprocessing library 'plutus-core-testlib' for plutus-core-1.1.0.0..
Building library 'plutus-core-testlib' for plutus-core-1.1.0.0..

<no location info>: error: [-Wunused-packages, -Werror=unused-packages]
    The following packages were specified via -package or -package-id flags,
    but were not needed for compilation:
      - data-default-class-0.1.2.0-CQYBH38PFES4dDyailJWvd
```

**Comment by zeme-iohk** [comment]
Some progress:
```
[17 of 28] Compiling PlutusCore.Generators.QuickCheck.Utils ( testlib/PlutusCore/Generators/QuickCheck/Utils.hs, dist/build/plutus-core-testlib>

testlib/PlutusCore/Generators/QuickCheck/Utils.hs:65:28: error: [-Wstar-is-type, -Werror=star-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: use ‘Type’ from ‘Data.Kind’ instead.
   |
65 |   data OneHoleContext f :: * -> *
   |                            ^

testlib/PlutusCore/Generators/QuickCheck/Utils.hs:65:33: error: [-Wstar-is-type, -Werror=star-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: use ‘Type’ from ‘Data.Kind’ instead.
   |
65 |   data OneHoleContext f :: * -> *
   |                                 ^
[25 of 28] Compiling PlutusIR.Generators.AST ( testlib/PlutusIR/Generators/AST.hs, dist/build/plutus-core-testlib/PlutusIR/Generators/AST.o, di>
[26 of 28] Compiling Test.Tasty.Extras ( testlib/Test/Tasty/Extras.hs, dist/build/plutus-core-testlib/Test/Tasty/Extras.o, dist/build/plutus-co>
```

**Comment by michaelpj** [comment]
I'm confused that you're finding _new_ errors here, shouldn't those have turned up on master already?

**Comment by zeme-iohk** [comment]
CI started failing 5 commits ago -> last 5 commits in master failed CI 
5th from last commit https://github.com/input-output-hk/plutus/pull/4880 was the one that introduced the new errors that we are finding here (see `plutus-core/testlib/PlutusCore/Generators/QuickCheck/Utils.hs`)

**Comment by zeme-iohk** [comment]
So to answer your question: they turned up but we ignored them or rather they were hidden by Cicero's "fail-fast" setting

**Comment by michaelpj** [comment]
I guess it was before we made it requied...

**Comment by zeme-iohk** [comment]
Yes. Also, I'm not sure the `ci` status (Required) is getting sent by Cicero. We'll see how this current invocation turns out

**Comment by zeme-iohk** [comment]
> Yes. Also, I'm not sure the `ci` status (Required) is getting sent by Cicero. We'll see how this current invocation turns out

I take it back. It kind of works: if all jobs pass, then `ci` goes green, but if any job fails, then `ci` stays in the `Expected` yellow state

**Comment by michaelpj** [comment]
Has a small conflict now

**Comment by zeme-iohk** [comment]
Keeping an eye on https://hydra.iohk.io/jobset/Cardano/plutus-flaked-pr-4926#tabs-evaluations
If that also passes then we can merge

**Comment by zeme-iohk** [comment]
> Keeping an eye on https://hydra.iohk.io/jobset/Cardano/plutus-flaked-pr-4926#tabs-evaluations
> If that also passes then we can merge

Actually I will need a new PR for that

**Pull request merged**